packages/cli/src/github/github.ts

```diff
@@ -156,21 +154,22 @@ export class Github {
    * Get all Github releases and match their tag names against the given semver
    * range. Return the release with the latest possible match.
    */
-  async getSemverRelease(semverRange: string): Promise<GitHubApi.Release> {
+  async getSemverRelease(semverRange: string): Promise<any> {
     // Note that we only see the 100 most recent releases. If we ever release
     // enough versions that this becomes a concern, we'll need to improve this
     // call to request multiple pages of results.
-    const releases: GitHubApi.Release[] = await this._github.repos.getReleases({
+    const response: Octokit.AnyResponse = await this._github.repos.getReleases({
       owner: this._owner,
       repo: this._repo,
       per_page: 100,
     });
+    const releases = response.data;
     const validReleaseVersions =
-        releases.filter((r) => semver.valid(r.tag_name)).map((r) => r.tag_name);
+        releases.filter((r: any) => semver.valid(r.tag_name)).map((r: any) => r.tag_name);
```

Comment:
Unfortunately, it seems like their `index.d.ts` no longer has types for responses.